src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -73,6 +73,10 @@ public string RegionOutcomeString
         {
             get => _regionOutcome.HasValue ? _regionOutcome.Value.ToString("D") : null;
         }
+        public string AutoDetectedRegion { get; set; }
```

Comment:
Can the sets be internal (if set only by this assembly) or private (if set only by this class)?